web/parser.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"fmt"
+	"github.com/PuerkitoBio/goquery"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func Parser() {
+	pageURL := "https://eais.rkn.gov.ru/"
```

Comment:
```suggestion

const pageURL = "https://eais.rkn.gov.ru/"

func Parser() {
```

We dont change it and it doesn't change so keep it a global constant incase we need to use it outside the Parser function